services/codecov/codecov.service.js

```diff
@@ -2,37 +2,34 @@
 
 const Joi = require('@hapi/joi')
 const { coveragePercentage } = require('../color-formatters')
-const { BaseJsonService } = require('..')
-
-// https://docs.codecov.io/reference#totals
-// A new repository that's been added but never had any coverage reports
-// uploaded will not have a `commit` object in the response and sometimes
-// the `totals` object can also be missing for the latest commit.
-// Accordingly the schema is a bit relaxed to support those scenarios
-// and then they are handled in the transform and render functions.
-const schema = Joi.object({
-  commit: Joi.object({
-    totals: Joi.object({
-      c: Joi.number().required(),
-    }),
-  }),
-}).required()
+const { BaseSvgScrapingService } = require('..')
 
 const queryParamSchema = Joi.object({
-  token: Joi.string(),
+  token: Joi.string().regex(/^\w{10}$/),
```

Comment:
Is there any documentation that states the possible valid token values?

If so it'd be great to include an inline comment with a link, but if not then I think we're better off leaving the schema as-is since it's just a pass through to the upstream service anyway and I'd prefer to not risk breaking any existing badges

Reply:
It appears as an error message when you use token of different length: https://codecov.io/gh/codecov/example-node/graph/badge.svg?token=abc

Other than that, there's no documentation link really.